egs/librispeech/ASR/pruned_transducer_stateless5/train.py

```diff
@@ -675,6 +691,7 @@ def compute_loss(
     info = MetricsTracker()
     with warnings.catch_warnings():
         warnings.simplefilter("ignore")
+        # info["frames"] is an approximate number
```

Comment:
perhaps should say why this is so? (presumably due to filtering inf/nan losses).
also, I suggest that if all of the either simple or pruned losses are infinite and the
batch has more than, say, 10 items, we could crash?  it's a waste of compute to 
continue to train a model that has diverged.  (The idea here is that 1 or 2 losses can
be infinite for valid reasons, but it's unlikely for a whole batch of 10; I'm picking a number
such that most setups will sometimes processa at least that many batches.  And it doesn't
require adding extra numbers to keep track off like the number of failures).

Reply:
Yes, will add it later today.

Reply:
Fixed. Please have a review again.

Sorry for the late reply.